### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,8 +29,55 @@ In your app settings::
...
+    {% block country %}
+        {% trans "Country" %}: {{request.site}}
+        <a href="{% url 'molo.globalsite:change_country' %}">{% trans "Change your country" %}</a>
+    {% endblock %}
+
+
+How users are redirected?
+=========================
+
+The middleware redirects the user by checking the session and if the country site is set in session it will redirect the user to the country site. If the country site is not in session and the geolocation is activated, it will uses user's IP address to detect their country and redirect the user to the supported country site. However if the detected country is not supported or the geolocation is not activated it will display the list of supported country sites.
```
Comment (Mitso) [inline README.rst]: it will uses to _it will use_

File: molo/globalsite/tests/test_views.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def setUp(self):
         asia = Region.objects.create(name='Asia')
 
         CountrySite.objects.create(
-            name='South Africa', code='ZA',
+            name='South Africa', code='za',
```
Comment (Mitso) [inline molo/globalsite/tests/test_views.py:23]: The the test for both lower case as upper country code?

**Review by Mitso — APPROVED** [review]
Small comments otherwise 🍰 🍰 🍰  from me

File: molo/globalsite/tests/test_views.py (hunk 2 above)
Comment (smarzban) [reply molo/globalsite/tests/test_views.py:23]: @Mitso the next country I'm creating it has the upper case country code but I will add specific test for it as well

**Pull request merged**